src/polhemus_tf_broadcaster.cpp

```diff
@@ -258,26 +259,27 @@ int main(int argc, char** argv) {
     for (i=0; i < 8; i++) {
       if (broadcast_stations[i]) {
 	s = i + 1;
-	static tf::TransformBroadcaster br;
-	tf::Transform transform;
-
-	// Set translation
-	transform.setOrigin(tf::Vector3(0.0254*stations[i].x,
-					0.0254*stations[i].y,
-					0.0254*stations[i].z));
-
-	// Set orientation
-	tf::Quaternion q;
-	q.setRPY(0.0174533*stations[i].ro,
-		 0.0174533*stations[i].el,
-		 0.0174533*stations[i].az);
-	transform.setRotation(q);
-
-	// Send tf
-	br.sendTransform(tf::StampedTransform(transform,
-					      ros::Time::now(),
-					      "polhemus_base",
-					      "polhemus_station_"+std::to_string(s)));
+	static tf2_ros::TransformBroadcaster br;
+	geometry_msgs::TransformStamped transformStamped;
+
+	// Header info
+	transformStamped.header.stamp = ros::Time::now();
+	transformStamped.header.frame_id = "polhemus_base";
+	transformStamped.child_frame = "polhemus_station_" + std::to_string(s);
+	
+	// Set translation (inches)
+	transformStamped.transform.translation.x = stations[i].x;
```

Comment:
ROS uses meters. So the conversion was okay but you need to document why and not just use a magic number